### ISSUE TITLE
Website should be accessible before ticket sales end

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [JoshuaKGoldberg/speaker-rider#6]
Right now I phrase accessibility timing as:

> Any website created by the event should conform to [WCAG 2.1](https://www.w3.org/WAI/WCAG21/quickref) by the date of the conference at Level A and at least mostly at AA.

...but what about folks trying to buy tickets?
